std/format/package.d

```diff
@@ -144,6 +144,59 @@ recommended to assign (lower- and uppercase) letters.
 Note: The $(I Parameters) of a $(I CompoundIndicator) are currently
 limited to a $(B '-') flag.
 
+$(SECTION4 Format Indicator)
+
+The $(I format indicator) can either be a single character or an
+expression surrounded by $(B %\() and $(B %\)). It specifies the
+basic manner in which a value will be formatted.
```

Comment:
```suggestion
expression surrounded by $(B %\() and $(B %\)). It is the minimum
requirement to format a value.
```
Perhaps ?

Reply:
Your sentence is a good idea, I'll add it. But I feel unwell with removing my suggestion - this sentence was meant to define the purpose of the format indicator, which IMHO would be missing if this sentence is removed.